docs/classifier_example/classification_v3_4_data_points.ipynb

```diff
@@ -0,0 +1,793 @@
+    "\\begin{equation}\n",
+    "    \\ket{\\mathcal{D}_D} = \\frac{1}{\\sqrt{8}} \\sum_{m=0}^{3} \\ket{m}\\Big(\\ket{0}\\ket{\\bf\\tilde{{x}}} + \\ket{1}\\ket{\\bf{x}_m}\\Big)\\ket{y_m}.\n",
+    "\\end{equation}\n",
+    "<img src=\"images/images4points/partd.png\">\n",
+    "\n",
+    "* **Part E:** In this part the actual distance-based classifier part of the algorithm is done. This part is independent of number of data points, which is precisely the strength of this quantum classifier. This step consists of a simple Hadamard-gate. Results are then obtained by post-processing the measurement results.\n",
+    "<img src=\"images/images4points/parte.png\">\n",
+    "\n",
+    "[Back to Table of Contents](#contents)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"implementation\"></a>\n",
+    "# Implementation #\n",
+    "We will implement the above algorithm using the projectQ framework. First we need to import some modules and set up the  authentication for connecting to the Quantum Inspire API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter email:\n",
+      "demo@quantum-inspire.com\n",
+      "Enter password\n",
+      "········\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Imports for authentication with Quantum Inspire API\n",
+    "from getpass import getpass\n",
+    "from coreapi.auth import BasicAuthentication\n",
+    "\n",
+    "# Import the projectQ backend from the Quantum Inspire\n",
+    "from quantuminspire.api import QuantumInspireAPI\n",
+    "from quantuminspire.projectq.backend_qx import QIBackend\n",
+    "\n",
+    "# Import projectQ \n",
+    "from projectq import MainEngine\n",
+    "from projectq.backends import ResourceCounter\n",
+    "from projectq.cengines import ManualMapper\n",
+    "from projectq.meta import Compute, Control, Loop, Uncompute\n",
+    "from projectq.ops import CNOT, All, H, Measure, Toffoli, X, Z, CZ, Ry, C\n",
+    "from projectq.setups import restrictedgateset\n",
+    "\n",
+    "\n",
+    "def get_authentication():\n",
+    "    \"\"\" Gets the authentication for connecting to the Quantum Inspire API.\"\"\"\n",
+    "    if 'password' not in vars().keys():\n",
+    "        print('Enter email:')\n",
+    "        email = input()\n",
+    "        print('Enter password')\n",
+    "        password = getpass()\n",
+    "    return BasicAuthentication(email, password)\n",
+    "\n",
+    "# Remote Quantum-Inspire backend #\n",
+    "authentication = get_authentication()"
```

Comment:
Should this not be more in line with the other notebooks:
```
token = load_account()
if token is not None:
    authentication = get_token_authentication(token)
else:
    if QI_EMAIL is None or QI_PASSWORD is None:
        ...
```
?

Reply:
Oke